### PR TITLE
updating the readme

### PR DESCRIPTION
**Pull request by cp16net**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+#### Login to the machine as root
+
+#### Make sure we have git installed:
+
+    $ apt-get update
+    $ apt-get install git-core -y
+    $ adduser ubuntu
+
+#### Add user to the sudoers:
```
Comment (rnirmal) [inline README.md]: Don't think this step is needed if you install a fresh copy... The default user you add during install will have sudo privs.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    $ ./redstack build-image
+
+#### Start up the reddwarf services in a screen session
+
+    $ ./redstack start
+
+***
+
+#### Running the reddwarf client (its so easy!)
+
```
Comment (rnirmal) [inline README.md]: Might be good to add that it automatically setups up the auth and local endpoints to hit.

File: README.md (hunk 1 above)
Comment (cp16net) [inline README.md]: sure its done on an install for us with vmware/vagrant but i did not notice it done on a cloud server when i set it up. Maybe some more verbiage around when this is needed?

File: README.md (hunk 2 above)
Comment (cp16net) [inline README.md]: Yeah thats a good point. We would also need to describe each of the commands in the list a little more than just the small little description that is printed with them. That can be done later though. This was just to get someone setup with the environment.

**Comment by cp16net** [comment]
Ok made a few of those updates.

**Comment by rnirmal** [comment]
+1.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    $ ./redstack start
+
+***
+
+#### Running the reddwarf client (its so easy!)
+*This sets of the authorization endpoint and gets a token for you*
+
+    $ ./redstack rd-client
+
+#### Running the nova client (its so easy!)
```
Comment (TimSimpsonR) [inline README.md]: S/b "it's". :p

**Comment by TimSimpsonR** [comment]
Great job! This looks awesome.
+1

**Pull request merged**